esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -80,11 +93,32 @@ var SchedulingCheck = React.createClass({
     } else if (!this.state.data) {
       body = <div className="placeholder">loading...</div>;
     } else {
+      var data = JSON.parse(this.state.data); // Might not work on old browsers
+      var table;
+      if (data.headings.length == 0) {
```

Comment:
This case and the other two I can find in this diff will be okay, but it's safer to more or less always use `===` instead of `==` in JS.  (See the [`==` equality table](https://dorey.github.io/JavaScript-Equality-Table/).)